src/pages/data-reporting/reports-api/v3.5/index.mdx

```diff
@@ -849,6 +849,70 @@ https://api.livechatinc.com/v3.5/reports/agents/performance \
 </Code>
 </Section>
 
+## Tags
+
+<Section>
+<Text>
+
+### Chat Usage
+
+Shows the total number of chats marked with each tag.
```

Comment:
Something's missing in this sentence. I guess we need a little more specificity as to what "each" refers to. E.g.:

> ...each tag within a license.

Reply:
more like "each tag listed in the report", wdyt? cause in the request you can specify groups to list tags from

Reply:
Sounds ok 👍🏻